packages/custom-properties/test/test.js

```diff
@@ -0,0 +1,47 @@
+import 'babel-polyfill'
```

Comment:
Is this needed for something? If so, we should probably fix it in the root-level Jest configuration

Reply:
Good question I will see what happens when I remove it. I was copying the `tachyons` package.

Reply:
Removed it without any problems.